test/beautify-stack.js

```diff
@@ -0,0 +1,44 @@
+var proxyquire = require('proxyquire').noPreserveCache();
+var test = require('tap').test;
+var beautifyStack = proxyquire('../lib/beautify-stack', {
+	debug: function () {
+		return {
+			enabled: false
+		};
+	}
+});
+
+test('does not strip ava internals and dependencies from stack trace with debug enabled', function (t) {
+	var beautify = proxyquire('../lib/beautify-stack', {
+		debug: function () {
+			return {
+				enabled: true
+			};
+		}
+	});
+
+	var result = beautify(
+		'Error: TypeError\n' +
+		'at null._onTimeout (/ava/cli.js:27:11)\n' +
+		'at Stub.listOnTimeout (timers.js:119:15)\n'
+	);
+
+	t.true(result.search('ava/cli.js') !== -1);
```

Comment:
This should use `indexOf` instead. You're actually not using a regex in the search() method.